src/boot/global-components.js

```diff
@@ -0,0 +1,18 @@
+//  global-components.js
+//
+//  Created by Kalila L. & Heather Anderson on May 13th, 2021.
+//  Copyright 2021 Vircadia contributors.
+//
+//  Distributed under the Apache License, Version 2.0.
+//  See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
+
+// Components
+import Audio from '../components/overlays/settings/Audio';
+import ChatWindow from '../components/overlays/chat/ChatWindow';
```

Comment:
Can `.vue` be included in filenames? To help make it easier to see that they're not JavaScript files.
Similarly, elsewhere.